pycobertura/cli.py

```diff
@@ -66,6 +66,13 @@ def get_exit_code(differ, source):
     type=str,
     help="Delimiter for csv format, e.g. ,;\n\t",
 )
+@click.option(
+    "-hide",
+    "--hidecolumns",
```

Comment:
Can we call this

```suggestion
    "--hide-columns",
```

... ?